.github/workflows/amis-workflow.yml

```diff
@@ -0,0 +1,157 @@
+---
+name: amis-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - 'v*'
```

Comment:
Can your replace with workflow dispatch as when we push to v1.*  packages are not published to production repo right away.

Reply:
thanks!.. sorry I am quite new to how the workflow dispatch works (I think I have only used it with inputs)
So.. will something like this be acceptable?

  workflow_dispatch:
    branches:
      - 'v1.*'

Reply:
just specify `workflow_dispatch: `, the ui will display a branch selector in github
you cannot specify filter on the branches, you would have to do it as a step if you want.